octopusdeploy/schema_machine_policy.go

```diff
@@ -129,9 +129,10 @@ func getMachinePolicyDataSchema() map[string]*schema.Schema {
 func getMachinePolicySchema() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"connection_connect_timeout": {
-			Default:  time.Minute,
-			Optional: true,
-			Type:     schema.TypeInt,
+			Default:     time.Minute,
+			Optional:    true,
+			Type:        schema.TypeInt,
+			Description: "In nanoseconds, no less than 10 seconds (10000000000)",
```

Comment:
This is the main change. See: https://octopusdeploy.slack.com/archives/C03SG0LFJHX/p1692940267454359

Reply:
I'd probably slightly adjust this to `In nanoseconds. Minimum value: 10000000000 (10 seconds).`